### PR TITLE
adds three blog posts, POT, Socialpoints and the IOT lamps

### PR DESCRIPTION
**Pull request by SkylerJaneClark**

### CODE REVIEW

File: _posts/2018-02-28-the-primary-owen-tracker.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: The Primary Owen Tracker
+date: 2018-02-28
+categories: projects
+description: The Primary Owen Tracker is a service for keeping track of the primary Owen of CSH. It allows people to suggest changes to the primary owen, and keep track of the current one. The rules for designating the primary owen is on Github!
```
Comment (csssuf) [inline _posts/2018-02-28-the-primary-owen-tracker.md]: s/owen/Owen

File: _posts/2018-02-28-the-primary-owen-tracker.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+author-bio: A freshman computer science student!
+author-email: sjc1689@rit.edu
+author-social:
+  github: https://github.com/puffballinthedark
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+
+The Primary Owen Tracker is a webapp I made that sprung up around a joke regarding who the "primary Owen" is, or in simpler terms, the Owen who most recently defeated the other Owen in a challenge. 
+
+It first started out as a web page on whoistheprimaryowen.tk, which has since been taken down. I registered that domain in about ten minutes of free time, and pointed it to a single HTML document on my CSH server with one header that said in plaintext who the primary owen was. However, somewhere along the line, I started adding things to the website, and ended up with a pretty involved project for an in-joke.
```
Comment (csssuf) [inline _posts/2018-02-28-the-primary-owen-tracker.md]: s/owen/Owen/

File: _posts/2018-02-28-the-primary-owen-tracker.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+author-social:
+  github: https://github.com/puffballinthedark
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+
+The Primary Owen Tracker is a webapp I made that sprung up around a joke regarding who the "primary Owen" is, or in simpler terms, the Owen who most recently defeated the other Owen in a challenge. 
+
+It first started out as a web page on whoistheprimaryowen.tk, which has since been taken down. I registered that domain in about ten minutes of free time, and pointed it to a single HTML document on my CSH server with one header that said in plaintext who the primary owen was. However, somewhere along the line, I started adding things to the website, and ended up with a pretty involved project for an in-joke. 
+
+The rules of the primary owen have evolved a bit since its first inception, but the gist of it is rougly the same.
```
Comment (csssuf) [inline _posts/2018-02-28-the-primary-owen-tracker.md]: s/rougly/roughly/

File: _posts/2018-02-28-the-primary-owen-tracker.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+
+The Primary Owen Tracker is a webapp I made that sprung up around a joke regarding who the "primary Owen" is, or in simpler terms, the Owen who most recently defeated the other Owen in a challenge. 
+
+It first started out as a web page on whoistheprimaryowen.tk, which has since been taken down. I registered that domain in about ten minutes of free time, and pointed it to a single HTML document on my CSH server with one header that said in plaintext who the primary owen was. However, somewhere along the line, I started adding things to the website, and ended up with a pretty involved project for an in-joke. 
+
+The rules of the primary owen have evolved a bit since its first inception, but the gist of it is rougly the same. 
+
+you can become the primary owen if you either are already named Owen (or Rowan, who was voted in as an Owen), and beat the current primary Owen in any kind of challenge, or if you are not already named Owen,  by winning a challenge against all existing Owens.
```
Comment (csssuf) [inline _posts/2018-02-28-the-primary-owen-tracker.md]: `s/^you/You/`
`s/,  /, /`

File: _posts/2018-02-28-the-primary-owen-tracker.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+I made the website in flask and hosted it on Openshift, using a mysql database hosted by CSH. I learned how to make webapps, use flask as a backend, and work with web interfaces. I also learned how to use mysql datbases, git for source control, and Openshift to host projects.
+
+You can find it on github right [here](https://github.com/puffballinthedark/primaryOwentracker)!
```
Comment (csssuf) [inline _posts/2018-02-28-the-primary-owen-tracker.md]: s/github/Github/

File: _posts/2018-02-28-socialpoints.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+author: Skyler Clark
+author-image: https://avatars1.githubusercontent.com/u/4897492?s=460&v=4
+author-bio: A freshman computer science student!
+author-email: sjc1689@rit.edu
+author-social:
+  github: https://github.com/puffballinthedark
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+
+Socialpoints is a website based around incrementing your own points while decrementing other CSHer's points. Essentially it is a cookie-clicker type game with no real goal other than to increment a counter, but it's a fun thing to play with and a fun project to build.
```
Comment (csssuf) [inline _posts/2018-02-28-socialpoints.md]: maybe "and was a fun project to build"?

File: _posts/2018-02-28-socialpoints.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+Socialpoints started out as a joke in the User Center, where we were just talking about how it would be great if there was a point system that could be arbitrarily assigned to CSHers for no reason other than to have points and take those points away. For some reason pretty much everyone in the room was in agreement that this would not only be a great idea, but also cause for a lot of mayhem, and I figured that it was too much fun not to build.
```
Comment (csssuf) [inline _posts/2018-02-28-socialpoints.md]: s/just//
s/cause for/cause/

File: _posts/2018-02-28-socialpoints.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+Socialpoints started out as a joke in the User Center, where we were just talking about how it would be great if there was a point system that could be arbitrarily assigned to CSHers for no reason other than to have points and take those points away. For some reason pretty much everyone in the room was in agreement that this would not only be a great idea, but also cause for a lot of mayhem, and I figured that it was too much fun not to build. 
+
+I started out designing my webapp with the knowledge that if I were to present CSHers with a button that could increment anything, they would take it, write scripts to automate the process, and crash my server. So one of the most important (and time consuming) things I had to do was set a cooldown for the "add points" button to prevent this outcome. 
+
+I ended up learning how to set up a webserver to host the site and how to set up a relational database to store and manage the data in the game, along with sharpening my HTML/Javascript/CSS skills and flask know-how. 
+
+All of the source for the project can be found [Here!](https://github.com/puffballinthedark/socialPoints)
```
Comment (csssuf) [inline _posts/2018-02-28-socialpoints.md]: s/Here/here/

File: _posts/2018-02-28-internet-of-lamps.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+author-bio: A freshman computer science student!
+author-email: sjc1689@rit.edu
+author-social:
+  github: https://github.com/SkylerJaneclark
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+
+The Internet of Lamps project is two linked lamps that communicate through a flask webapp to turn on simultaneously so long as they're connected to WiFi. 
+
+I started this project wondering what I should make for my girlfriend for a christmas present, and started browsing the web for ideas. I ended up coming across a cheap IOT board called the nodeMCU which gave me the idea of building a couple lamp, and decided to just go in blind.
```
Comment (csssuf) [inline _posts/2018-02-28-internet-of-lamps.md]: s/christmas/Christmas/
s/couple lamp/couple lamps/

File: _posts/2018-02-28-internet-of-lamps.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+
+The Internet of Lamps project is two linked lamps that communicate through a flask webapp to turn on simultaneously so long as they're connected to WiFi. 
+
+I started this project wondering what I should make for my girlfriend for a christmas present, and started browsing the web for ideas. I ended up coming across a cheap IOT board called the nodeMCU which gave me the idea of building a couple lamp, and decided to just go in blind. 
+
+Figuring out how electricity actually worked was probably the hardest part of the project. I ended up frying one of my boards entirely by running current through it backwards, and set an ultrasonic sensor on fire by doing the same thing. I had to figure out how to get power to a light strip while sending less power to the board, how to program the board (and get it to work with my girlfiend's wifi 7 hours away), how I was going to handle the requests the boards were sending on my webserver, laser cut the actual wooden part of the lamp, and how to do all that before christmas came, with no experience having even touched anything similar!
```
Comment (csssuf) [inline _posts/2018-02-28-internet-of-lamps.md]: s/christmas/Christmas/

File: _posts/2018-02-28-internet-of-lamps.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+Luckily I managed, and now we have a little way of saying hello to each other from all the way across new england.
```
Comment (csssuf) [inline _posts/2018-02-28-internet-of-lamps.md]: s/new england/New England/

File: _posts/2018-02-28-the-primary-owen-tracker.md (hunk 5 above)
Comment (RamZallan) [reply _posts/2018-02-28-the-primary-owen-tracker.md]: Looks like it's technically [`GitHub`](https://en.wikipedia.org/wiki/GitHub), although as long as it's capitalized, that works too imo.

File: _posts/2018-02-28-socialpoints.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+  - projects
+description: Socialpoints is a webapp for raking CSHers by their ability to click buttons fast. As a result, socialpoint has become a webapp for making bots that can click faster than everyone elses' bots. Built on flask and based around relational databses.
+image: https://csh.rit.edu/~wgahnagl/heyyyyo.png
+image-sm: https://csh.rit.edu/~wgahnagl/primaryOwenSmaller.png
+author: Skyler Clark
+author-image: https://avatars1.githubusercontent.com/u/4897492?s=460&v=4
+author-bio: A freshman computer science student!
+author-email: sjc1689@rit.edu
+author-social:
+  github: https://github.com/puffballinthedark
```
Comment (RamZallan) [inline _posts/2018-02-28-socialpoints.md]: It looks like you changed your GH username, can you update the link to it?

**Comment by RamZallan** [comment]
In addition to the requested changes, could you bump the date up on this post (both filename and metadata)?
